cql/CDSiSupportingData-4.3.cql

```diff
@@ -0,0 +1,192 @@
+library CDSiSupportingData version '4.3'
+
+using FHIR version '4.0.0'
+using CDSI version '1.0.0'
+include FHIRHelpers version '4.0.0' called FHIRHelpers
+
+parameter AntigenSupportingData CDSI.AntigenSupportingData
+
+parameter AntigenAncillaryData Tuple{series List<Tuple{seriesName String, primarySeriesNumberDoses Integer}>}
+
+context Patient
+
+define "Immunizations":
+	[Immunization] immunization
+	sort by (occurrence as FHIR.dateTime) asc
+
+define "Run":
+	EvaluateAllSeries
+
+define "EvaluateAllSeries":
+	AntigenSupportingData.series series
+		let evaluations: "EvaluateSeries"(series.seriesName, series.selectSeries, series.seriesDose)
+		where series.seriesType != 'Risk'	
```

Comment:
Since we're including `Evaluation Only` seriesType that means a "2-dose AstraZeneca Series" or other Non-FDA authorized vaccination series will be recognized as valid.

I'm ok with that, but we should document this somewhere (README or another markdown file in the repo?). Some of the UI tickets will need to consider this when describing what a valid vaccination series means since it doesn't align with some of the CDC sites (e.g. https://www.cdc.gov/coronavirus/2019-ncov/vaccines/stay-up-to-date.html `When Are You Up to Date` only mentions the FDA authorized vaccines)

Reply:
That site is intended for US vaccination forecasting, but doesn't apply to foreign travelers to US. The CDC does recognize these non-US cases as valid vaccination series as long as they are WHO accepted vaccines, which is why they're included in the official guidance. We can note this in the FAQs.

Reply:
Created DVCI-266 to help document this for developers/implementors who might want to implement their own CQL.